crates/tools/msvc/src/main.rs

```diff
@@ -67,7 +67,7 @@ fn main() {
             break;
         }
     }
-
+    archive.flush();
```

Comment:
May want to call `drop(archive)` right after this line to ensure the file is closed after flushing and before copying.

Reply:
I think the gnu tool does call `drop` but it fails to call `flush`.